### PR TITLE
Add contact-manager code samples

### PR DESCRIPTION
**Pull request by lisatr2812**

### CODE REVIEW

File: contact-manager/index.mjs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+)
+
+
+//tech-doc: Create a contact with services
+const serviceToCreate = await contactApi.service().newInstance(
+    loggedUser,
+    new Service({
+        valueDate: 20211020,
+        status: 0,
+        codes: [new CodeStub({
```
Comment (lisatr2812) [inline contact-manager/index.mjs:44]: Attention : Content & codes not added in sent dto

Comment (aduchate) [reply contact-manager/index.mjs:44]: Why is that ?

File: contact-manager/index.mjs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            id: 'http://loinc.org|16935-9|0',
```
Comment (aduchate) [inline contact-manager/index.mjs:45]:
```suggestion
            id: 'http://loinc.org|16935-9|2',
```

File: contact-manager/index.mjs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            context: 'code',
+            system: 'http://loinc.org',
+            code: '16935-9',
+            version: '0',
```
Comment (aduchate) [inline contact-manager/index.mjs:49]:
```suggestion
            version: '2',
```

**Pull request merged**